dts/bindings/led_strip/worldsemi,ws2812-spi.yaml

```diff
@@ -33,3 +38,8 @@ properties:
     type: int
     required: true
     description: 8-bit SPI frame to shift out for a 0 pulse.
+
+  spi-empty-preamble-frames:
+    type: int
+    default: 0
+    description: Number of empty frames to send before the data burst.
```

Comment:
Since both the SPI frequency and the reset delay are known from the driver, then the preamble frame size can be computed without any other input.

Reply:
That is a good point, let me rework this, just part way through adding the kconfig to toggle the whole thing off.